proto/exocore/delegation/v1/query.proto

```diff
@@ -69,6 +70,28 @@ message SingleDelegationInfoReq {
   string asset_id = 3 [(gogoproto.customname) = "AssetID"];
 }
 
+// UndelegationsReq is the request to obtain all delegations
+// by staker id and asset id.
+message UndelegationsReq {
+  // staker_id is the staker id.
+  string staker_id = 1 [(gogoproto.customname) = "StakerID"];
+  // asset_id is the asset id.
+  string asset_id = 2 [(gogoproto.customname) = "AssetID"];
+}
+
+// UndelegationsReq is the request to obtain all undelegations waiting to be completed
+// by height.
+message WaitCompleteUndelegationsReq {
```

Comment:
can we use PendingUndelegationsReq ?

Reply:
Maybe, using `UndelegationsByHeightReq` is better, because the difference with `UndelegationsReq` is that it queries the undelegations by block height. I will rename it to `UndelegationsByHeightReq`.